.github/workflows/test.yml

```diff
@@ -219,6 +218,28 @@ jobs:
             exit 1
           fi
 
+  shell:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Test shell"
+        id: test-shell
+        uses: ./
+        with:
+          shell: 'container | from alpine | with-exec echo,"hello world!" | stdout'
+      - name: "Test shell (check)"
+        run: |
+          target='${{ steps.test-shell.outputs.output }}'
+          result='hello world!
+          ' # shell appears to emit an extra newline compared to call
```

Comment:
Ah, I suspect this is because `echo` includes a newline (while the call to `shykes/daggerverse` just directly returns a string without the newline).

We could either use the same test for both, *or* just use `-n` on the `echo`.